src/mod_bosh.erl

```diff
@@ -174,7 +174,7 @@ init(Req, _Opts) ->
 info(accept_options, Req, State) ->
     Origin = cowboy_req:header(<<"origin">>, Req),
     Headers = ac_all(Origin),
-    ?DEBUG("OPTIONS response: ~p~n", [Headers]),
+    ?LOG_DEBUG(#{options_response => Headers}),
```

Comment:
OK, what field is required.

```
?LOG_DEBUG(#{what => bosh_accept_options, text => <<"Handle OPTIONS request in Bosh">>, headers => Headers})
```

Reply:
I will fix it in all other log maps too.